tests/test_utils/test_schemaview.py

```diff
@@ -34,998 +31,872 @@
 RELATED_TO = 'related to'
 AGE_IN_YEARS = 'age in years'
 
+@pytest.fixture
+def schema_view_no_imports():
+    return SchemaView(SCHEMA_NO_IMPORTS)
+
+
+@pytest.fixture
+def view():
+    """Fixture for SchemaView with imports."""
+    return SchemaView(SCHEMA_WITH_IMPORTS)
 
-class SchemaViewTestCase(unittest.TestCase):
 
-    def test_children_method(self):
+def test_children_method():
         view = SchemaView(SCHEMA_NO_IMPORTS)
```

Comment:
You don't need to redefine `view` in all these test functions since you made a fixture for it. Just include `view` or `schema_view_no_imports` as an argument to the function.

Reply:
yep, makes sense.  Some of the tests redefine the import to test components of importing but not this.  Fixed (I replaced the new SchemaView initialization with the fixture but left the `view` variable assignment in place because not doing so would mean many more test changes.)